website/source/docs/state/remote/s3.html.md

```diff
@@ -56,3 +56,6 @@ The following configuration options / environment variables are supported:
  * `access_key` / `AWS_ACCESS_KEY_ID` - (Optional) AWS access key
  * `secret_key` / `AWS_SECRET_ACCESS_KEY` - (Optional) AWS secret key
  * `kms_key_id` - (Optional) The ARN of a KMS Key to use for encrypting the state.
+ * `profile` - (Optional) This is the AWS profile name as set in the shared credentials file.
+ * `shared_credentials_file`  - (Optional) This is the path to the shared credentials file. If this is not set and a profile is specified, ~/.aws/credentials will be used.
+ * `token` - (Optional) Use this to set an MFA token. It can also be sourced from the `AWS_SECURITY_TOKEN` environment variable.
```

Comment:
This is not really an MFA token, it's session token and can be used any time user uses `AssumeRole*` API - no matter if it's MFA-protected or isn't, but that's really a nitpick, I can fix that in a separate commit.